### PR TITLE
[Enhancement] #27 Opengraph meta tags

### PR DESCRIPTION
**Pull request by anxolerd**
# Enhancement #27 Opengraph meta tags

Added opengraph meta tags to the following templates:
- [event.html](https://github.com/anxolerd/gdg.org.ua/commit/c73064ed7465703dc5142b0c16256b5be23c33ee)
- [register.html](https://github.com/anxolerd/gdg.org.ua/commit/6344acdeb30d3ed67cb1fdf53ecf5b49bc3927bb)
- [gdg.org.ua_old.html](https://github.com/anxolerd/gdg.org.ua/commit/0965473adac7a216ce94f0823900ba13acfe3bdc)

### CODE REVIEW

**Comment by anxolerd** [comment]
Review request: @webknjaz

File: src/GDGUkraine/templates/event.html [hunk 1]
```diff
@@ -9,10 +9,21 @@
...
     <meta name="author" content="Valentyn Shybanov">
+
+    <!-- Start opengraph meta -->
+    <!-- Override author, cause opengraph picks last author meta -->
+    <meta name="author" content="{{ event.host_gdg.name or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine' }}">
+    <!-- Actually opengraph meta -->
+    <meta property="og:title" content="{{ event.title }}">
+    <meta property="og:description" content="{{ event.description }}">
+    <meta property="og:site_name" content="GDG Ukraine Events">
+    <meta property="og:image" content="{{event.host_gdg.logo or '/img/gdg.png' if event.host_gdg_id else '/img/gdg.png' }}">
```
Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: No space after `{{`

File: src/GDGUkraine/templates/event.html [hunk 2]
```diff
@@ -9,10 +9,21 @@
     <meta name="author" content="Olexandr Kovalchuk">
     <meta name="author" content="Svyatoslav Sydorenko">
     <meta name="author" content="Valentyn Shybanov">
+
+    <!-- Start opengraph meta -->
+    <!-- Override author, cause opengraph picks last author meta -->
+    <meta name="author" content="{{ event.host_gdg.name or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine' }}">
```
Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: Are the strings well-escaped when inserted?

File: src/GDGUkraine/templates/event.html (hunk 1 above)
Comment (anxolerd) [inline src/GDGUkraine/templates/event.html]: fixed in 0fae3d9

File: src/GDGUkraine/templates/event.html (hunk 2 above)
Comment (anxolerd) [inline src/GDGUkraine/templates/event.html]: Hm. I thought automatic escaping was enabled for project. I was wrong.

fixed in a54ad36 and 15ecdbe

**Comment by webknjaz** [comment]
Plz implement Twitter Card as follows https://github.com/douhack/douhack/blob/gh-pages/public/index.html#L10-L15
I think [this one](https://dev.twitter.com/cards/types/summary-large-image) would fit well.
Validator: https://cards-dev.twitter.com/validator

Also special prefixes may have to be added. i.e. `<html prefix="og: ...`

**Comment by anxolerd** [comment]
Added twitter metatags in 3d8ab48, 15ffe6b, a5175ba

**Comment by anxolerd** [comment]
Added namespace prefixes in 3baa2c6, 1a069a4, 1235da5

File: src/GDGUkraine/templates/event.html [hunk 3]
```diff
@@ -9,10 +9,28 @@
...
     <meta name="author" content="Valentyn Shybanov">
+
+    <!-- Start opengraph meta -->
+    <!-- Override author, cause opengraph picks last author meta -->
+    <meta name="author" content="{{ event.host_gdg.name|e or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine' }}">
+    <!-- Actually opengraph meta -->
+    <meta property="og:title" content="{{ event.title|e }}">
+    <meta property="og:description" content="{{ event.desc|e }}">
+    <meta property="og:site_name" content="GDG Ukraine Events">
+    <meta property="og:image" content="http://gdg.org.ua/img/pattern.png"><!-- TODO: refactor with dynamic url generation -->
```
Comment (webknjaz) [inline src/GDGUkraine/templates/event.html]: Plz use https

**Comment by anxolerd** [comment]
Chaned http to https in 8541e97

**Pull request merged**